<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>scene</class>
 <widget class="QMainWindow" name="scene">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1850</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLCDNumber" name="countDown">
    <property name="geometry">
     <rect>
      <x>1700</x>
      <y>30</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QProgressBar" name="expBar">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>35</y>
      <width>1600</width>
      <height>15</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QProgressBar" name="HP_bar">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>230</y>
      <width>75</width>
      <height>15</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">	border: 1px solid grey;
	border-radius: 3px;
	background-color: transparent;
	text-align: center;
	qproperty-textVisible: false;
	height: 8px;</string>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>220</y>
      <width>75</width>
      <height>75</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelb">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>140</y>
      <width>75</width>
      <height>75</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="boss_hpbar">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>350</y>
      <width>75</width>
      <height>15</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">	border: 1px solid grey;
	border-radius: 3px;
	background-color: transparent;
	text-align: center;
	qproperty-textVisible: false;
	height: 8px;</string>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1850</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menucoin">
    <property name="title">
     <string>coin:</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu0">
    <property name="title">
     <string>0</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>|               |</string>
    </property>
   </widget>
   <widget class="QMenu" name="menulevel">
    <property name="title">
     <string>exp:</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu02">
    <property name="title">
     <string>0</string>
    </property>
   </widget>
   <widget class="QMenu" name="menulevel_2">
    <property name="title">
     <string>|               |</string>
    </property>
   </widget>
   <widget class="QMenu" name="menulevel_3">
    <property name="title">
     <string>level:</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu1">
    <property name="title">
     <string>1</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>|               |</string>
    </property>
   </widget>
   <widget class="QMenu" name="menurec">
    <property name="title">
     <string>rec:</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu3">
    <property name="title">
     <string>3</string>
    </property>
   </widget>
   <addaction name="menucoin"/>
   <addaction name="menu0"/>
   <addaction name="menu"/>
   <addaction name="menulevel"/>
   <addaction name="menu02"/>
   <addaction name="menulevel_2"/>
   <addaction name="menulevel_3"/>
   <addaction name="menu1"/>
   <addaction name="menu_2"/>
   <addaction name="menurec"/>
   <addaction name="menu3"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
